include/utility/vpd_specific_utility.hpp

```diff
@@ -466,5 +466,88 @@ inline bool findCcinInVpd(const nlohmann::json& i_JsonObject,
     logging::logMessage("VPD type not supported. Can't find CCIN");
     return false;
 }
+
+/**
+ * @brief API to clear VPD of the FRU on DBus
```

Comment:
Please move this API to a new PR and call it at the beginning of primeInventory so that it can address defect https://github.com/ibm-openbmc/openpower-vpd-parser/issues/419 as well.